docs/platform/howto/create_new_service_user.rst

```diff
@@ -3,6 +3,15 @@
 
 Service users are users that only exist in the scope of the corresponding Aiven service. They are unique to this service and not shared with any other services, and can be granted restricted permissions compared to the default ``avnadmin`` user. You can add service users for all the Aiven services, with the exception of Aiven for Apache Flink® and Aiven for Grafana®.
 
+.. warning::
+   By default, the maximum amount of users allowed on a service is 50. If a service user is created that exceeds the max users, the following error will be produced.
+   
+   .. code-block:: python 
+      
+      "Service user 'xxx' cannot be created, maximum number of users for this plan exceeded"
```

Comment:
```suggestion
```

Reply:
Just removing the error code. As it is shown in console, we don't need to document it here.

Reply:
Works for me. :)

Reply:
For this one, my main goal was to provide and example error and explanation for customers getting this error, the hope being that they would be better informed on what to ask for when they open a case with us. For instance, as opposed to getting cases like "I got this error, what do I do do" we would get cases like  "Request to increase max users for service". Let me know your thoughts on that.

Reply:
Thanks for explaining. I see where you are coming from with this. In general, we try to avoid documenting things in the UI like this just because it can also easily go out of date with changes. Maybe we could just reword to say:
 "If you try to create too many service users, you will receive an error message stating, "maximum number of users for this plan exceeded".

This way we could also remove the code block and keep that formatting for code, but the key words will still be available for a search. Let me know if that works!